tests/models/test_modeling_common.py

```diff
@@ -1436,6 +1437,7 @@ def test_fn(storage_dtype, compute_dtype):
         test_fn(torch.float8_e5m2, torch.float32)
         test_fn(torch.float8_e4m3fn, torch.bfloat16)
 
+    @is_flaky
```

Comment:
If it's only for `test_models_autoencoder_hunyuan_video.py`, I would consider subclassing it inside `AutoencoderKLHunyuanVideoTests` and decorating it with `is_flaky`.

Reply:
These tests haven't been running on CI for a while due to the unclip issue so it is potentially flaky for other models too.

Reply:
Would prefer confirming that first before marking all tests as flaky when they appear to be not.